### PR TITLE
Add version 2019-03-01-preview Microsoft.ServiceFabric specification and fix package-2018-02 mismatched duplicate models

### PR DESCRIPTION
**Pull request by a-santamaria**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/6090 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/6090/commits/7a3a91a90edd23e6f161262c8746d9f97d15a557

File: specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-03-01-preview/application.json [hunk 1]
```diff
@@ -0,0 +1,2133 @@
...
+          "Operations"
+        ],
+        "summary": "Lists all of the available Service Fabric resource provider API operations.",
+        "description": "Get the list of available Service Fabric resource provider API operations.",
+        "operationId": "Operations_List",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "The version of the Service Fabric resouce provider API",
```
Comment (nschonni) [inline specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-03-01-preview/application.json]:
```suggestion
            "description": "The version of the Service Fabric resource provider API",
```

File: specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-03-01-preview/application.json [hunk 2]
```diff
@@ -0,0 +1,2133 @@
...
+            "description": "The application resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApplicationResource"
+            },
+            "x-ms-parameter-location": "method"
+          }
+        ],
+        "x-ms-examples": {
+          "Put an application with mininum parameters": {
```
Comment (nschonni) [inline specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-03-01-preview/application.json]:
```suggestion
          "Put an application with minimum parameters": {
```

File: specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-03-01-preview/application.json [hunk 3]
```diff
@@ -0,0 +1,2133 @@
...
+            "description": "The service resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceResource"
+            },
+            "x-ms-parameter-location": "method"
+          }
+        ],
+        "x-ms-examples": {
+          "Put a service with mininum parameters": {
```
Comment (nschonni) [inline specification/servicefabric/resource-manager/Microsoft.ServiceFabric/preview/2019-03-01-preview/application.json]:
```suggestion
          "Put a service with minimum parameters": {
```

**Comment by sarangan12** [comment]
@a-santamaria Apologize for the delay. Please fix the errors reported at https://dev.azure.com/azure-sdk/public/_build/results?buildId=30903&view=logs&jobId=688669d0-441c-57c3-cf6d-f89a22ccfa5d

**Comment by a-santamaria** [comment]
> @a-santamaria Apologize for the delay. Please fix the errors reported at https://dev.azure.com/azure-sdk/public/_build/results?buildId=30903&view=logs&jobId=688669d0-441c-57c3-cf6d-f89a22ccfa5d

@sarangan12 thanks for taking a look. the error reported is DuplicateModelCollsion saying that file: /Microsoft.ServiceFabric/preview/2017-07-01-preview/application.json has a definition of ApplicationHealthPolicy in line 973. But as you can see [here](https://github.com/Azure/azure-rest-api-specs/pull/5852/files#diff-c718aac16be257fcf7ae8ea8f9b4f162L973) I removed that definition from the file so I'm not sure why the check is still reporting that error.

**Comment by sarangan12** [comment]
@a-santamaria  Let me run some checks and update this PR

**Comment by sarangan12** [comment]
This is definitely failing SDK generation. I am still debugging this. Will update soon.

**Comment by a-santamaria** [comment]
@sarangan12  any update on this? Is there an error in my changes that is causing this?

**Review by amarzavery — APPROVED** [review]

**Pull request merged**